static/500.html

```diff
@@ -0,0 +1,207 @@
+  p { font-size: 1.1rem; }
+}
+@media (min-width: 700px) {
+  h1 {font-size: 5.0rem;}
+  h6 {font-size: 1.2rem;}
+  p {font-size: 1.3rem;} 
+}
+
+p {
+  font-family: 'Roboto', sans-serif;
+  color: #000000;
+  margin-top: 0; 
+  font-weight: 300; 
+}
+
+
+code {
+  vertical-align: middle; 
+  padding: 0.1rem 0.1rem;
+  margin-left: auto;
+  margin-right:auto;
+  white-space: nowrap;
+  background: #F1F1F1;
+  border: 1px solid #E1E1E1;
+  border-radius: 0px; }
+pre > code {
+  display: block;
+  padding: 0.5rem 0.5rem;
+  white-space: pre; }
+
+span {
+  text-indent: 1.5em; 
+  margin-left: 
+  width: 100%; 
+}
+
+
+
+
+  /* Self Clearing Goodness */
+.container:after,
+.row:after,
+.u-cf {
+  content: "";
+  display: table;
+  clear: both; }
+
+
+
+
+  </style>
+
+
+  <!-- Add Gazelle Favicon 
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <link rel="icon" type="image/png" href="./favicons/favicon-16x16.png">
+
+</head>
+<body>
+  <header>
+    <div class="header column, headerLogo">
+        <svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+             viewBox="0 0 208.000000 208.000000"
+             preserveAspectRatio="xMidYMid meet">
+
+            <g transform="translate(0.000000,208.000000) scale(0.100000,-0.100000)"
+            fill="#000000" stroke="none">
+            <path d="M628 2013 c-44 -82 -47 -124 -41 -487 l6 -338 -24 7 c-13 4 -59 25
+            -102 47 -119 59 -319 139 -341 136 -28 -4 -39 -56 -34 -164 4 -102 3 -101 132
+            -238 54 -58 55 -59 103 -54 26 3 66 13 88 23 22 10 69 30 105 45 l65 27 6
+            -166 c5 -136 10 -177 28 -226 12 -33 86 -181 164 -330 l142 -270 112 -3 112
+            -3 46 78 c26 43 101 187 168 321 l122 243 3 180 c2 98 5 179 6 179 2 0 55 -22
+            119 -50 74 -32 131 -50 156 -50 37 0 45 6 118 86 43 47 83 95 90 107 13 23 18
+            204 7 245 -4 13 -14 22 -25 22 -26 0 -202 -71 -337 -136 -62 -30 -117 -54
+            -122 -54 -6 0 -9 140 -7 373 l2 372 -31 59 c-17 33 -35 61 -41 63 -6 2 -84
+            -368 -191 -902 -100 -498 -185 -911 -189 -918 -5 -9 -97 437 -373 1821 -2 13
+            -22 -7 -42 -45z m141 -1098 c17 -27 39 -86 50 -130 53 -223 132 -587 129 -596
+            -2 -6 -57 101 -123 238 l-120 248 -3 170 -2 170 19 -25 c11 -13 33 -47 50 -75z
+            m626 68 c-2 -16 -6 -91 -9 -168 l-5 -140 -123 -247 c-67 -136 -124 -246 -126
+            -243 -2 2 3 37 12 77 139 617 142 629 209 709 38 46 48 49 42 12z"/>
+            </g>
+            </svg>
+      </div>
+      <!--  <hr style="width:100%; height:2px; background-color: #000000; border:none;" />
+       -->
+
+  </header>
+
+  <!-- Primary Page Layout
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <div class="container">
+    <div class="row">
+      <div style="margin-top: 5%">
+        <h1>Shoot...</h1>
+        <pre><code><h6><span style ="color: #FF4242">Error Code: </span>500</h6> <!-- <h6 class="fontErr2">console.log(<span style = "color:#FF4242">"Abort: Server broken"</span>);<br/>
```

Comment:
I like this just as `Error Code: 500`. It won't confuse non-techy people as much as the `console.log(...)`